README.md

```diff
@@ -1 +1,73 @@
-# Hell_Microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Hello Microverse project
+
+
+## Built With
+
+- html-css
+
+## Live Demo (if available)
+
+
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
```

Comment:
-[ ] Update this by explaining to the users on how can they clone and run your project